### PR TITLE
Feat: Basic structure for build-rainer component

### PR DESCRIPTION
**Pull request by Tailp**
Hi :) , related issue (#899)

### CODE REVIEW

File: repairnator/build-rainer/pom.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <groupId>fr.inria.repairnator</groupId>
+  <artifactId>build-rainer</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <name>build-rainer</name>
+  <url>http://maven.apache.org</url>
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>3.8.1</version>
```
Comment (monperrus) [inline repairnator/build-rainer/pom.xml]: depend on junit4 or even better junit5

File: repairnator/build-rainer/src/main/java/fr/inria/repairnator/Launcher.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package fr.inria.repairnator;
+
+public class Launcher {
+    public static void main( String[] args ) {
```
Comment (monperrus) [inline repairnator/build-rainer/src/main/java/fr/inria/repairnator/Launcher.java]: put the main in class BuildRainer directly

File: repairnator/build-rainer/src/main/java/fr/inria/repairnator/BuildRainer.java [hunk 3]
```diff
@@ -0,0 +1,6 @@
+package fr.inria.repairnator;
+
+public class BuildRainer {
```
Comment (monperrus) [inline repairnator/build-rainer/src/main/java/fr/inria/repairnator/BuildRainer.java]: add one sentence of javadoc

File: repairnator/build-rainer/pom.xml (hunk 1 above)
Comment (Tailp) [reply repairnator/build-rainer/pom.xml]: I'm on it immediately :)

File: repairnator/build-rainer/src/main/java/fr/inria/repairnator/Launcher.java (hunk 2 above)
Comment (Tailp) [reply repairnator/build-rainer/src/main/java/fr/inria/repairnator/Launcher.java]: Fixed.

File: repairnator/build-rainer/pom.xml (hunk 1 above)
Comment (Tailp) [reply repairnator/build-rainer/pom.xml]: Fixed.

File: repairnator/build-rainer/src/main/java/fr/inria/repairnator/BuildRainer.java (hunk 3 above)
Comment (Tailp) [reply repairnator/build-rainer/src/main/java/fr/inria/repairnator/BuildRainer.java]: Fixed.

**Comment by monperrus** [comment]
please don't answer "fixed" click on "resolve".

**Comment by Tailp** [comment]
> 
> please don't answer "fixed" click on "resolve".

Yes :) .

**Comment by monperrus** [comment]
Thanks a lot @Tailp !

**Comment by Tailp** [comment]
Thank you :), I'll start implementing this.

**Pull request merged**